js-api/src/chem.ts

```diff
@@ -150,6 +155,7 @@ export namespace chem {
     resized = false;
     _sketcherTypeChanged = false;
     _autoResized = true;
+    _validationFunc: ((molecule: string) => IValidationRes) | null = null;
```

Comment:
Probably easier to define it as non-nullable and initialize at place

_validationFunc: ((molecule: string) => IValidationRes) = ((s) => null);